src/optional-listeners/message-command-listeners/CorePreMessageCommandRun.ts

```diff
@@ -12,15 +12,15 @@ export class CoreListener extends Listener<typeof Events.PreMessageCommandRun> {
 
 		// Run global preconditions:
 		const globalResult = await this.container.stores.get('preconditions').messageRun(message, command, payload as any);
-		if (!globalResult.success) {
-			message.client.emit(Events.MessageCommandDenied, globalResult.error, payload);
+		if (!globalResult.isErr()) {
```

Comment:
```suggestion
		if (!globalResult.isOk()) {
```
Bug

Reply:
We'll want to rewrite this to not check is err but use one of the error methods to mapinto the error object. Nonetheless great find.